apps/api/.env.example

```
@@ -5,6 +5,7 @@ DATABASE_URL="postgresql://postgres:[YOUR-PASSWORD]@db.[YOUR-PROJECT-REF].supaba
 SUPABASE_URL="https://[YOUR-PROJECT-REF].supabase.co"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)